### PR TITLE
Cleanup Gradle

### PR DESCRIPTION
**Pull request by Elforama**
Updated gradle version
Added ViewBindings to eventually replace synthetics usage
Removed dependencies that stopped gradle upgrade
Cleaned things up

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,20 +1,7 @@
-buildscript {
-    repositories {
-        maven { url 'https://maven.fabric.io/public' }
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'io.fabric.tools:gradle:1.28.0'
-        classpath 'com.github.triplet.gradle:play-publisher:1.2.0'
```
Comment (arunbakt) [inline app/build.gradle:9]: Are we not depending on play publisher? What is taking its place?
I assume crashlytics is taking over fabric correct?

Comment (arunbakt) [inline app/build.gradle:9]: Are we not depending on play publisher? What is taking its place?
I assume crashlytics is taking over fabric correct?

File: app/build.gradle [hunk 2]
```diff
@@ -31,15 +18,10 @@ def loadExtraProperties(String fileName) {
 
 loadExtraProperties("treetracker.keys.properties")
 
-
 android {
     compileSdkVersion 29
     buildToolsVersion '29.0.0'
-    playAccountConfigs {
-        defaultAccountConfig {
-            jsonFile = file('keys.json')
```
Comment (arunbakt) [inline app/build.gradle:40]: I forgot what this was meant for.

**Review by arunbakt — COMMENTED** [review]
It is nice that we are upgrading the version on gradle. I guess all the tools you removed are obstructing the upgrade, am I right?

File: app/build.gradle (hunk 1 above)
Comment (Elforama) [reply app/build.gradle:9]: I'll double check of play publisher, I don't think it's used.

yes, firebase is replacing fabric

File: app/build.gradle (hunk 2 above)
Comment (Elforama) [reply app/build.gradle:40]: This is for the play console plugin I think.

**Pull request merged**